Index: server/account/admin.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from django.contrib import admin\r\n\r\n# Register your models here.\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server/account/admin.py b/server/account/admin.py
--- a/server/account/admin.py	(revision a12e783ce5d050d1aaecffc8d8fa1bac23cafb24)
+++ b/server/account/admin.py	(date 1703062097320)
@@ -1,3 +1,49 @@
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from account.forms import CustomUserCreationForm, CustomUserChangeForm
+from account.models import User
 
-# Register your models here.
+
+@admin.register(User)
+class UserAdmin(UserAdmin):
+    fieldsets = (
+        (None, {"fields": ('username', "email", "password", 'first_name', 'last_name')}),
+        (
+            "Permissions",
+            {
+                "fields": (
+                    "is_staff",
+                    "is_active",
+                    "is_superuser",
+                    "groups",
+                    "user_permissions",
+                )
+            },
+        ),
+        ("Dates", {"fields": ("last_login", "date_joined")}),
+        # ('required', {
+        #          'fields': ('first_name', 'last_name')})
+    )
+    add_fieldsets = (
+        (
+            None,
+            {
+                "classes": ("wide",),
+                "fields": (
+                    "email",
+                    "password1",
+                    "password2",
+                ),
+            },
+        ),
+    )
+    add_form = CustomUserCreationForm
+    form = CustomUserChangeForm
+
+    model = User
+
+    ordering = ["email"]
+
+    list_display = ('id', 'username', 'email', 'first_name', 'last_name')
+    list_display_links = ('id', 'username')
Index: server/account/forms.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server/account/forms.py b/server/account/forms.py
new file mode 100644
--- /dev/null	(date 1700602079391)
+++ b/server/account/forms.py	(date 1700602079391)
@@ -0,0 +1,17 @@
+from django.contrib.auth.forms import UserChangeForm, UserCreationForm
+
+from .models import User
+
+
+class CustomUserCreationForm(UserCreationForm):
+
+    class Meta:
+        model = User
+        fields = ("email",)
+
+
+class CustomUserChangeForm(UserChangeForm):
+
+    class Meta:
+        model = User
+        fields = ("email",)
